acceptancetests/assess_remove_machine.py

```diff
@@ -0,0 +1,85 @@
+import argparse
```

Comment:
Missing hash bang, although not strictly required, we should include them, until all of the acceptance tests are migrated to python3.

Reply:
Good point, have added.